main/REPL/priceAuthority.md

```diff
@@ -0,0 +1,191 @@
+# Price Authority and Timer Service
+
+`PriceAuthority` objects use the timer service. Therefore, before covering `PriceAuthority`, we must
+first look at the timer service's objects and methods.
+
+## Timer Service
+
+`TimerService` objects have methods for getting the current time, scheduling
+a `wake()` call, scheduling repeated events, and cancelling scheduled events.
+
+### `getCurrentTimestamp()`
+- Returns: `{ Timestamp }`
+
+Returns the last timestamp.
+
+### `setWakeup(baseTime, waker)`
+- `baseTime` `{ Timestamp }`
+- `waker` `{ TimerWaker }`
+- Returns: `{ Timestamp }`
+
+Returns the time when the call is scheduled to happen
+
+### `removeWakeup(waker)`
+- `waker` `{ TimerWaker }`
+- Returns: `{ Array<Timestamp> }`
+
+Remove the `waker` from all scheduled wakeups, whether produced by `timer.seWakeup(h)`
+or `repeater.schedule(h)`.
+
+### `createRepeater(delay, interval)`
+- `delay` `{ RelativeTime }`
```

Comment:
Agoric/agoric-sdk#2143 will change the first param's name to `delaySecs`. Might as well change it now.